distribution/docs/src/main/resources/content/_running/manual-removal-expired.adoc

```diff
@@ -0,0 +1,38 @@
+:title: Manual Removal of Expired Records
+:type: dataManagement
+:status: published
+:summary: Methods of removing Expired Records from the Catalog.
+:parent: Removing expired records from the Catalog
+:order: 00
+
+== {title}
+
+To manually remove expired records from the Catalog, execute in the ${command-console}:
+
+----
+catalog:removeall --expired
+----
+
+When prompted, type `yes` to remove all expired records.
+
+[TIP]
+====
+For help on the `removeall` command, execute
+
+`help catalog:removeall`
+====
+
+The `catalog:removeall` command states you want to remove records from the local Catalog.
+
+The `--force` option is used to suppress the confirmation message which asks a user if the user intentionally wants to permanently remove records from the Catalog.
```

Comment:
I think everyone has a pretty good idea of what "force" means. It is used in a lot of commands. I suggest replacing the explanation of `--force` with the same explanation provided in `help catalog:removeall`:
" Force the removal without a confirmation message."

Reply:
fair enough. done